acra-censor/handlers/blacklist_handler.go

```diff
@@ -88,9 +89,10 @@ func (handler *BlacklistHandler) CheckQuery(query string) (bool, error) {
 	}
 	//Check patterns
 	if len(handler.patterns) != 0 {
+		//fmt.Println("here")
```

Comment:
`git add -p acra-censor/handlers/blacklist_handler.go` and you can not include this :)